modules/ui/OptimizerParamsWindow.py

```diff
@@ -129,6 +129,18 @@ def create_dynamic_ui(
             'warmup_init': {'title': 'Warmup Initialization', 'tooltip': 'Whether to warm-up the optimizer initialization.', 'type': 'bool'},
             'weight_decay': {'title': 'Weight Decay', 'tooltip': 'Regularization to prevent overfitting.', 'type': 'float'},
             'weight_lr_power': {'title': 'Weight LR Power', 'tooltip': 'During warmup, the weights in the average will be equal to lr raised to this power. Set to 0 for no weighting.', 'type': 'float'},
+            'decoupled_decay': {'title': 'Decoupled Decay', 'tooltip': 'If set as True, then the optimizer uses decoupled weight decay as in AdamW.', 'type': 'bool'},
```

Comment:
We already have an option called "decouple" from prodigy that is meant to be this; just reuse that?

Reply:
At the moment, all parameter names map directly to the optimizer parameter names. Unless we want to change that, adding an extra parameter would be the right choice I think.

Reply:
I think we should change that, it's going to be difficult to help people who come in with settings json files if two optimizers have similar sounding and acting parameters with each name in the json. I already have problems with bias correction and use bias correction helping debug people's problems.